tests/code/test_program_eligibility.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from app.models.eventParticipant import EventParticipant
+from app.controllers.events.volunteerRegisterEvent import volunteerRegister
+from peewee import DoesNotExist
+from app.models.user import User
+from app.models.event import Event
+from app.controllers.events.programEligibility import isEligibleForProgram
+from app.models.program import Program
+@pytest.mark.integration
+def test_noUserVolunteerRegister():
+
+    with pytest.raises(DoesNotExist):
+        eligible = isEligibleForProgram(2, "asdlkfje") #user doesn't exist
+
+    with pytest.raises(DoesNotExist):
+        eligible = isEligibleForProgram(2, 135156)
+
+    with pytest.raises(DoesNotExist):
+        eligible = isEligibleForProgram(9, "lamichhanes2") #Program doesn't exist
+
+
+
+@pytest.mark.integration
+def test_volunteerEligible():
```

Comment:
You don't test any failure cases. You need to test all of the logic in your function. When would this function return False?

Reply:
Added more test cases where the function returns False